src/client/sandbox/child-window/index.ts

```diff
@@ -124,6 +106,31 @@ export default class ChildWindowSandbox extends SandboxBase {
         });
     }
 
+    _handleClickOnLinkWithTargetBlankAttr (window: Window): void {
+        if (!settings.get().allowMultipleWindows)
+            return;
+
+        window.addEventListener('click', e => {
```

Comment:
Use a native method for `addEventListener` here.
@LavrovArtem if it's possible add this method to `eslint-plugin-hammerhead`.

Reply:
this PR is still in DRAFT. I discussed this approach with @LavrovArtem and we decided to modify it

Reply:
Ok

Reply:
ready, please review